Resources/Prototypes/GameRules/second_life.yml

```diff
@@ -0,0 +1,115 @@
+- type: entity
+  id: LifeTourists
+  parent: BaseGameRule
+  noSpawn: true
+  components:
+  - type: StationEvent
+    weight: 10
+    startDelay: 10
+    duration: 120
+    earliestStart: 20
+    reoccurrenceDelay: 20
+  - type: SecondLifeRule
+    announce: second-life-tourist-announce
+    description: second-life-tourist
+    roles:
+      Passenger: 15
```

Comment:
overall with these: a few too many roles to give out imo. 15 passengers is definitely too many. ideally it'd be random from like 5-10, i feel like? if it stays static, i'd prefer for it to be at 7 or 8 in total